examples/Catena4430_Sensor/Catena4430_cMeasurementLoop_SDcard.cpp

```diff
@@ -125,9 +125,9 @@ cMeasurementLoop::checkSdCard()
     }
 
 static const char kHeader[] =
-    "Time,DevEUI,Raw,Vbat,Vsystem,Vbus,BootCount,T,RH,P,Light,"
```

Comment:
Leave an empty column for vsystem. In fact, for clarity, leave the Vsystem label.